enterprise/views.py

```diff
@@ -499,38 +505,65 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_uuid, course
             logger.error('Unable to find course details for course ID: %s', course_id)
             raise Http404
 
-        platform_name = configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)
+        try:
+            enrollment_client = EnrollmentApiClient()
+            modes = enrollment_client.get_course_modes(course_id)
+        except HttpClientError:
+            logger.error('Failed to determine available course modes for course ID: %s', course_id)
+            raise Http404
+
+        # This is protected by the enterprise_login_required decorator; we don't need to guard it here
+        enterprise_customer = EnterpriseCustomer.objects.get(uuid=enterprise_uuid)  # pylint: disable=no-member
+
+        course_modes = []
+
+        audit_modes = getattr(settings, 'ENTERPRISE_COURSE_ENROLLMENT_AUDIT_MODES', ['audit'])
```

Comment:
Is there any reason anyone knows of why we wouldn't include `honor` in the list of audit modes by default? Even though it's no longer used by edx.org, it's still in the Open edX platform, and it is a free (non-premium) mode.